src/lib.rs

```diff
@@ -386,9 +388,47 @@ impl Installer {
                     }
                 };
 
+                // Takes the locale, such as `en_US.UTF-8`, and changes it into `en_US`.
+                let locale = match config.lang.find('.') {
```

Comment:
We actually need to get `en` from `en_US.UTF-8`. It may be better not to pass locale, so that it keeps all language packs. I believe this will be desired for most Ubuntu installations.

`check-language-support --language=en_US --show-installed` - all packages for English United States
`check-language-support --language=en --show-installed` - all packages for every English country
`check-language-support --show-installed` - all packages for all installed languages. This appears to be equivalent to the following languages on Ubuntu: `de en es fr it pt ru zh zh-hans zh-hant`